visualisations/fan_chart/webviz_fan_chart/__init__.py

```diff
@@ -0,0 +1,148 @@
+from webviz_plotly import Plotly
+import pandas as pd
+import matplotlib.cm as cm
+
+color_scheme = cm.get_cmap('Set1')
+
+
+def color_spread(n):
+    colorlist = []
+    for i in range(n):
+        single_color = color_scheme(float(i) / n)
+        formatted_color = []
+        for y in single_color[:-1]:
+            formatted_color.append(str(y))
+        colorlist.append(formatted_color)
+    return colorlist
+
+
+def format_color(color, opacity):
+    return "({},{})".format(','.join(color), opacity)
+
+
+def init_scatter_trace(y, mean, x, name, line, color, index):
+    return [{
```

Comment:
I prefer that init_scatter_trace only returns a dictionary, not a list. Simpler interface. At the invocation you can use list.append.

Reply:
Also this function should have a docstring.